### PR TITLE
Following users

### PR DESCRIPTION
**Pull request by ducminhvu**

### CODE REVIEW

File: app/views/users/_unfollow.html.erb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<%= form_for current_user.active_relationships.find_by(followed_id: @user.id),
+      html: { method: :delete }, remote: true do |f| %>
```
Comment (NguyenTanDuc) [inline app/views/users/_unfollow.html.erb]: remove space { method: :delete } => {method: :delete}

File: app/controllers/relationships_controller.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
+class RelationshipsController < ApplicationController
+  before_action :logged_in_user
+
+  def index
+    type = params[:type]
+    if type == "following"
+      @title = t :following
+      @user  = User.find params[:id]
+      @users = @user.following.paginate page: params[:page]
```
Comment (phamkykhoi) [inline app/controllers/relationships_controller.rb]: insert per_page: Settings

File: app/controllers/relationships_controller.rb [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    else
+      @title = t :followers
+      @user  = User.find params[:id]
+      @users = @user.followers.paginate page: params[:page]
```
Comment (phamkykhoi) [inline app/controllers/relationships_controller.rb]: insert per_page: Settings

File: app/controllers/relationships_controller.rb [hunk 4]
```diff
@@ -0,0 +1,36 @@
+class RelationshipsController < ApplicationController
+  before_action :logged_in_user
+
+  def index
+    type = params[:type]
+    if type == "following"
```
Comment (phamkykhoi) [inline app/controllers/relationships_controller.rb]: Move following to Settings

File: app/controllers/relationships_controller.rb [hunk 5]
```diff
@@ -0,0 +1,36 @@
+class RelationshipsController < ApplicationController
+  before_action :logged_in_user
+
+  def index
```
Comment (phamkykhoi) [inline app/controllers/relationships_controller.rb]: I want move script show follower and following to index action in users_controller

File: app/controllers/relationships_controller.rb [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+    send params[:type]
```
Comment (phamkykhoi) [inline app/controllers/relationships_controller.rb]: An error occurred when hack param on url

File: app/views/relationships/create.js.erb [hunk 7]
```diff
@@ -0,0 +1,2 @@
+$("#follow_form").html("<%= escape_javascript(render('users/unfollow')) %>");
```
Comment (phamkykhoi) [inline app/views/relationships/create.js.erb]: ' ' => " "

File: app/views/relationships/destroy.js.erb [hunk 8]
```diff
@@ -0,0 +1,2 @@
+$("#follow_form").html("<%= escape_javascript(render('users/follow')) %>");
```
Comment (phamkykhoi) [inline app/views/relationships/destroy.js.erb]: ' ' => " "

File: app/views/relationships/destroy.js.erb [hunk 9]
```diff
@@ -0,0 +1,2 @@
+$("#follow_form").html('<%= escape_javascript render "users/follow" %>');
```
Comment (phamkykhoi) [inline app/views/relationships/destroy.js.erb]: ' ' => " "

**Pull request merged**